api/Window.json

```diff
@@ -2338,24 +2338,35 @@
       "gamepadconnected_event": {
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Window/gamepadconnected_event",
-          "spec_url": "https://w3c.github.io/gamepad/#the-gamepadconnected-event",
+          "spec_url": [
+            "https://w3c.github.io/gamepad/#the-gamepadconnected-event",
+            "https://w3c.github.io/gamepad/#event-gamepadconnected"
+          ],
           "description": "<code>gamepadconnected</code> event",
           "support": {
             "chrome": [
               {
-                "version_added": "35"
+                "version_added": "35",
+                "partial_implementation": true,
+                "notes": "The event handler property (<code>ongamepadconnected</code>) is not supported; see <a href='https://crbug.com/1192878'>bug 1192878</a>."
               },
               {
                 "version_added": "21",
                 "version_removed": "35",
-                "prefix": "webkit"
+                "prefix": "webkit",
```

Comment:
I tried to verify this to check if there was a `onwebkitgamepadconnected` but it turns out there were never prefixed events. The unprefixed event were here:
https://chromium.googlesource.com/chromium/src/+/8634dbe6397b6271278792a6f42b3be2c03e8013

Going by https://www.chromium.org/developers/calendar/ that's Chromium 35, so I think this entry and any mirrored ones should just be deleted.